### PR TITLE
Replace Drafts logic with Nova Drafts field

### PR DESCRIPTION
**Pull request by KasparRosin**
Added [nova-drafts](https://github.com/optimistdigital/nova-drafts) field to replace integrated drafts logic.
**All functionality will remain.**

`NovaBlog::draftsEnabled()` is replaced with `NovaBlog::hasNovaDrafts`.
Removed `drafts_enabled` from config. Now it checks whether user has the field package installed.
Removed most of draft logic, that was replaced by nova-drafts

### CODE REVIEW

File: config/nova-page-manager.php [hunk 1]
```diff
@@ -13,18 +13,6 @@
...
-    /*
-    |--------------------------------------------------------------------------
-    | Drafts enabled
-    |--------------------------------------------------------------------------
-    |
-    | If set to true, drafting capabilities will be available.
-    |
-    */
-
-    'drafts_enabled' => false,
```
Comment (Tarpsvo) [inline config/nova-page-manager.php:25]: Draftid ei ole optional enam?

File: src/Models/Page.php [hunk 2]
```diff
@@ -86,7 +54,7 @@ public function isDraft()
     public function getPathAttribute()
     {
         if (!isset($this->parent)) return $this->normalizePath($this->slug);
-
+
```
Comment (Tarpsvo) [inline src/Models/Page.php]: Ebavajalik tab

File: src/NovaPageManager.php [hunk 3]
```diff
@@ -28,9 +28,9 @@ public function renderNavigation()
         return view('nova-page-manager::navigation');
     }
 
-    public static function draftsEnabled(): bool
+    public static function hasNovaDrafts(): bool
     {
-        return config('nova-page-manager.drafts_enabled', false);
+        return class_exists('\OptimistDigital\NovaDrafts');
```
Comment (Tarpsvo) [inline src/NovaPageManager.php]: Kas siin ei peaks olema` \OptimistDigital\NovaDrafts\NovaDrafts`?

**Review by Tarpsvo — CHANGES_REQUESTED** [review]

File: config/nova-page-manager.php (hunk 1 above)
Comment (KasparRosin) [reply config/nova-page-manager.php:25]: Ma tegin [sarnase funktsiooni nagu NovaLangiga on tehtud](https://github.com/optimistdigital/nova-page-manager/blob/28b7067ef8e1c708f73f87091b4a6254c264a8b3/src/NovaPageManager.php#L31)
Kasutaja peab composeriga installima nova-draftid

File: src/NovaPageManager.php [hunk 4]
```diff
@@ -28,9 +28,9 @@ public function renderNavigation()
         return view('nova-page-manager::navigation');
     }
 
-    public static function draftsEnabled(): bool
+    public static function hasNovaDrafts()
     {
-        return config('nova-page-manager.drafts_enabled', false);
+        return class_exists('\OptimistDigital\NovaDrafts\DraftButton') || class_exists('\OptimistDigital\NovaDrafts\PublishedField');
```
Comment (Tarpsvo) [inline src/NovaPageManager.php:33]: `||` asemel peaks olema `&&`, ei?

**Review by Tarpsvo — APPROVED** [review]

**Pull request merged**